### DISCUSSION TITLE
Add circle size alignment setting to skin.ini

### DISCUSSION THREAD

**Discussion by ThatBebx** [ppy/osu#21806] [Ideas] [answered]
Add a circle size alignment setting to the skin.ini. 

The hit circle image would be scaled up in-game by a factor provided in the skin.ini file, so that there is more room to work with on hitcircle.png and hitcircleoverlay.png files. This would allow for effects like drop shadows, glows, and other visual elements outside of the hitbox of a hitcircle. This would not effect the hitbox, just the visual scale of the hit circle. Retroactively this setting would default to 1.

An implementation could look like:
HitCircleOverlayScale:2 
(scales the hit circle overlay image ingame up by a factor of 2, so that the user has double the space on the hitcircleoverlay.png to draw on)

Personally I want this feature so that I can play with the background dim lowered. Drop shadows would make the circles easier to read in this situation, but larger shadows aren't really possible with the current skinning scheme for hit circles (you currently only have about 4 pixels to work with). (lighting.png does not fulfill this purpose either!!)

Hopefully this was clearly enough worded. Thanks for reading! I understand if this is slightly out of scope, but I think is a sensible inclusion (well, I mean, of course I would, it's my suggestion 😜, but still.)

**Comment by peppy** [comment, marked as answer]
This likely won't be supported as it would allow unfairly creating larger objects / visibility.
